src/com/company/BloomFilter/BloomFilterImpl.java

```diff
@@ -0,0 +1,66 @@
+package com.company.BloomFilter;
+
+import sun.security.util.BitArray;
+
+/**
+ * An implementation of the BloomFilter algorithm.
+ */
+public class BloomFilterImpl implements BloomFilter {
+    private final int size;
+    public static final int DEFAULT_SIZE = 32;
+    private final BitArray bits;
+
+    public BloomFilterImpl() {
+        size = DEFAULT_SIZE;
```

Comment:
This will look a bit prettier:

```suggestion
        this.size = DEFAULT_SIZE;
        this.bits = new BitArray(size);
```